src/pages/Cart/Cart.jsx

```diff
@@ -26,7 +26,7 @@ export const Cart = () => {
                         <h3>Your Cart Is Empty</h3>
                     </div> 
                     
-                    : mycart?.map(({ _id, imgSrc, title, type, price, inStock, fastDelivery, onSale }) => (
+                    : mycart?.map(({ _id, imgSrc, title, type, price, discount, inStock, fastDelivery, onSale, qty }) => (
                         <Fragment>
```

Comment:
<> works same as <Fragment>. I know this one is a little opinionated but you can pick anything from both.